spec/models/provider_spec.rb

```diff
@@ -571,4 +571,81 @@
       it { should_not include last_years_course }
     end
   end
+
+  describe "geolocation" do
```

Comment:
any chance to move this whole block to its own file
` spec/models/provider/geolocation.rb`

this `provider_spec.rb` is growing

Reply:
Yeah unfortunately we have had to get rid of the geolocation model concern therefore the geolocation specs had to move back into the model specs